### PR TITLE
Add JASPAR2020

### PR DESCRIPTION
**Pull request by da-bar**
Added compatibility with JASPAR2020

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -40,7 +40,9 @@ Suggests: BiocStyle(>= 1.7.7),
           JASPAR2014(>= 1.4.0),
           knitr(>= 1.11),
           testthat,
-          JASPAR2016(>= 1.0.0)
+          JASPAR2016(>= 1.0.0),
+          JASPAR2018(>= 1.0.0),
+          JASPAR
```
Comment (ge11232002) [inline DESCRIPTION]: The new data package should be JASPAR2020

Comment (da-bar) [reply DESCRIPTION]: Aziz wanted all the versions in a single package, so I made it that way. I can also make JASPAR2020 separately

Comment (ge11232002) [reply DESCRIPTION]: The thing is you are not supposed to update the data package, i.e., adding stuff to a existing data package.

Comment (da-bar) [reply DESCRIPTION]: Ok. I will convert it to JASPAR2020 at first convenience sometime during the day.

**Pull request merged**